src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -52,6 +52,8 @@ public CommandResult execute(Model model) {
             return new CommandResult(String.format(LIST_TAGS_MESSAGE_SUCCESS, allTags));
 
         } else {
+            model.updateToDefaultSortedTaskList();
+            model.updateToDefaultFilteredTaskList();
             model.updateFilteredTaskList(PREDICATE_SHOW_ALL_TASKS);
```

Comment:
Since we now use `updateToDefaultFilteredTaskList()`, will we still need `updateFilteredTaskList(PREDICATE_SHOW_ALL_TASKS)`?

Reply:
Good catch, I missed this. Thanks for spotting it!